### ISSUE TITLE
Features without location data

### ISSUE THREAD

**Issue by docuracy** [LinkedPasts/linked-places-format#41]
LP standard at present requires `coordinates` within geometries. The GeoJSON standard allows for unlocated features but requires that they have _null_ geometry.

PROPOSAL: Drop the requirement for `coordinates` to allow for standard GeoJSON representation of unlocated features. This will be useful not only for the validation of datasets where features have yet to be located, but also for datasets where location might be inappropriate (see [this proposal](https://github.com/LinkedPasts/linked-places-format/issues/39#issue-1149516362)).

**Comment by kgeographer** [comment]
The language in the spec is an interpretation of Section 3.1 of [the GeoJSON spec](https://datatracker.ietf.org/doc/html/rfc7946). Open to other interpretations!

> A GeoJSON Geometry object of any type other than "GeometryCollection" has a member with the name "coordinates". The value of the "coordinates" member is an array.  The structure of the elements in this array is determined by the type of geometry.  GeoJSON processors MAY interpret Geometry objects with empty "coordinates" arrays as null objects.

**Comment by rsimon** [comment]
Yes, it's not super clear IMO.... But a key point here is that the spec speaks about the `geometry` object. We've been representing `features`s without a geometry by giving them a `geometry` object with empty coordinates. That didn't work. (I guess _some_ parser MAY accept them, as the spec suggests.) The safe way - which is hinted here - is to represent unlocated features as `feature` objects where the `geometry` is a `null`.

**Comment by kgeographer** [comment]
Just tested use of `null` for a `geometry` in a Leaflet L.geoJSON layer, and it also is fine -- simply not rendered. I am editing the LP spec accordingly.
